README.md

```diff
@@ -28,11 +28,76 @@ You can grab latest binaries [here](https://ci.appveyor.com/project/olsh/curl-to
 1. Install cake
 
 ```bash
-dotnet tool install -g Cake.Tool`
+dotnet tool install -g Cake.Tool
 ```
 
 2. Run build
 
 ```bash
 dotnet cake build.cake
 ```
+
+## NuGet Packages
+### Curl.Parser.Net
+#### Key Features
+- Parse cURL command into individual cURL options.
+- Return parsing errors and warnings if the cURL input is invalid.
+
+#### Installation
+Install with NuGet
+```cmd
+dotnet add package Curl.Parser.Net
```

Comment:
We should update the package names in readme too

Reply:
Done